### ISSUE TITLE
Play integrity will no longer bypass

### ISSUE THREAD

**Issue by rom4ster** [kdrag0n/safetynet-fix#281]
As of today APR 17th, play integrity no longer bypasses and wallet payments dont work

**Comment by doblasds** [comment]
Same as here, can't pay in wallet.

**Comment by Displax** [comment]
https://github.com/Displax/safetynet-fix/releases/tag/v2.4.0-MOD_1.2

**Comment by doblasds** [comment]
> https://github.com/Displax/safetynet-fix/releases/tag/v2.4.0-MOD_1.2

I have installed and it doesn't work :S

**Comment by rom4ster** [comment]
> https://github.com/Displax/safetynet-fix/releases/tag/v2.4.0-MOD_1.2

this worked for me, I had to uninstall the normal one, I have no magiskhideprops (on newer magisk) and I have shamiko and dillia tweaks. 

DISPLAX I remember you. I downloaded from you once before, and then you pushed an update saying I no needed you and you pushed me out of your version into kdrag0ns. 

Could you just keep your version up to date this time instead ty :)

**Comment by jfeise** [comment]
> https://github.com/Displax/safetynet-fix/releases/tag/v2.4.0-MOD_1.2

Works for me, after uninstalling the original one.

**Comment by Gh61** [comment]
Remove the previous, restart and install moded version.
SafetyNet working again! Thanks

**Comment by Fede2782** [comment]
Same issue on A22 5G (A226B). Safetynet and Play Certification pass. However Play Integrity api doesn't pass, only BASIC_INTEGRITY and not DEVICE_INTEGRITY. I know that STRONG INTEGRITY will never pass on unlocked bootloader however DEVICE_INTEGRITY always did.

**Comment by Fede2782** [comment]
Fixed with mod by displax

**Comment by Pheggas** [comment]
Yup, I agree, this mod fixed the contactless payment (a.k.a Google Wallet)

**Comment by knasiotis** [comment]
@Displax mod does not work anymore in Xiaomi EU Rom stable 14.0.9.0
I uninstalled and reinstalled it and google pay fails sometimes, like before the mod.

**Comment by Displax** [comment]
> @Displax mod does not work anymore in Xiaomi EU Rom stable 14.0.9.0
> I uninstalled and reinstalled it and google pay fails sometimes, like before the mod.

Use version 2.3.1_mod-3.0

@knasiotis

**Comment by knasiotis** [comment]
Thank you! It works!

**Comment by baptx** [comment]
@Displax I did not get your fork working either for Android apps on my computer:
https://github.com/Displax/safetynet-fix/issues/28#issuecomment-1895647402

**Comment by rom4ster** [comment]
Use this https://github.com/chiteroman/PlayIntegrityFix

**Comment by baptx** [comment]
@rom4ster I tried both latest https://github.com/chiteroman/PlayIntegrityFix and https://github.com/osm0sis/PlayIntegrityFork on Waydroid but they did not work with the app Play Integrity API Checker (I added all apps and system apps to MagiskHide list). Some people said it is not possible to bypass SafetyNet / Play Integrity on emulated Android. Did you get it working for Android apps on a computer?

**Comment by knasiotis** [comment]
@baptx  you have to find your own fingerprint with PlayIntegrityFix, if you do it's possible to bypass safetynet

**Comment by baptx** [comment]
@knasiotis How can we find and use our own fingerprint? I don't have a real Android device by the way, I just use Waydroid on my computer.

**Comment by knasiotis** [comment]
you have to search for various android dumps scattered throughout the internet, and place that information in pif.json, there are guides on [xda](https://xdaforums.com/t/module-play-integrity-fix-safetynet-fix.4607985/)

**Comment by mrvictory1** [comment]
Does play integrity fork still work? I tried with magisk v26 + zygisknext + playintegrityfork + oneplus 6 and galaxy j4 fingerprints, can't pass basic or cts integrity.

**Comment by jfeise** [comment]
> Does play integrity fork still work? I tried with magisk v26 + zygisknext + playintegrityfork + oneplus 6 and galaxy j4 fingerprints, can't pass basic or cts integrity.

Google changed things. They now require a signed build, and blacklisted certain signatures. See https://github.com/chiteroman/PlayIntegrityFix I am using that fork.

**Comment by mrvictory1** [comment]
@jfeise what is your software configuration? Magisk version, modules + versions etc. I don't know if I installed something incorrectly or need a different fingerprint. I am using a custom built image (for a different reason), do I need to sign it? I renamed testkey.x509.pem to releasekey.x509.pem inside /system/etc/security/otacerts.zip without touching its contents.

EDIT: I'm using waydroid, I thought I was asing on waydroid github. Sorry.
